config/jobs/kubernetes/test-infra/test-infra-presubmits.yaml

```diff
@@ -65,3 +65,26 @@ presubmits:
         - prow/cluster
     annotations:
       testgrid-create-test-group: 'true'
+  - name: pull-test-infra-prow-checkconfig
+    branches:
+      - master
+    decorate: true
+    cluster: build
+    run_if_changed: '^config/prow/(config|plugins).yaml$'
+    spec:
+      containers:
+      - image: gcr.io/k8s-prow/checkconfig:v20191219-b8438ff62
```

Comment:
Will the autobump keep this in sync when we bump a version?

Reply:
Autobump appears to work for everything in `gcr.io/k8s-prow/`, so this image should get bumped even if this is our first time using it. [Source](https://github.com/kubernetes/test-infra/blob/4235df2f20e580db7fde8a53129d2bb90e97a337/prow/cmd/autobump/bump.sh#L79)